root/README.md

```diff
@@ -1,60 +1,18 @@
-﻿# Open Touryo
-*Open Touryo* is an application framework based on .NET Framework.  
-The programs of Open Touryo are available in the following repositories:
-- OpenTouryoTemplates repository
-    - Summary  
-    The programs in this repository are the *development infrastructure (project template)* of the programs using Open Touryo.  
-    Default DBMS is *SQL Server*, and the programs are partitioned into the folders for each version of Visual Studio.
-    - Intended User  
-    The *application developers* using Open Touryo.
-- OpenTouryo repository (*Current repository*)
-    - Summary  
-    The programs in this repository are the *matrix* of OpenTouryoTemplates repository.  
-    (First, the features of Open Touryo are implemented in this repository. And then, the features are introduced into OpenTouryoTemplates repository.)
-    - Intended User  
-    *Open source developers*.
-
-Therefore, the users who use Open Touryo in a system development project need to use [OpenTouryoTemplates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates).  
-The following contents are the usage of Open Touryo for *open source developers*.
+# Open Touryo Template Base for Visual Studio 2015
```

Comment:
@daisukenishino 

This repository is *NOT* template base.
Please do not replace this file with Readme file of template base.